src/formGenerator.vue

```diff
@@ -301,10 +310,16 @@ div.vue-form-generator(v-if='schema != null')
 			// Validating the model properties
 			validate() {
 				this.clearValidationErrors();
+				this.touched.splice(0);
 
 				this.$children.forEach((child) => {
 					if (isFunction(child.validate))
 					{
+						this.touched.push({
```

Comment:
Ok so your solution doesn't work.
When `validateAfterLoad` is true, the loop will trigger `getFieldRowClasses` before any field component initialization (`created` or `mounted`) and thus `fieldData` is empty, causing an error.
I think a solution like this would need a big rewrite of VFG internal.
I'm sorry but I cannot allow myself to lose any more time on this. I'm going to implement the fixes you suggested but I will keep this "dirty" for now.
Hope you understand, thank you for your help and feel free to do another PR with a better solution.